user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security.md

```diff
@@ -0,0 +1,116 @@
+---
+uid: Database_security
+---
+
+# Securing the DataMiner databases
+
+## Cassandra general database (default)
+
+### Authentication
+
+By default, DataMiner installs Cassandra with the PasswordAuthenticator enabled. Cassandra comes installed with a default *cassandra* user. When DataMiner installs Cassandra (e.g. during migration or installation), an extra superuser named *root* is created.
+
+#### Configuring more secure passwords for the default users
+
+We highly recommend that you configure more secure passwords for the default user. Preferably, these passwords should be randomly generated and stored in a password vault.
+
+You can do so by executing the following queries (using DevCenter, the DataMiner Cube Query Executor, or your preferred query tool):
+
+`ALTER USER cassandra WITH PASSWORD '<NEW PASSWORD>'`
+
+`ALTER USER root WITH PASSWORD '<NEW PASSWORD>'`
+
+#### Creating a new superuser and removing the default user
+
+We also recommend that you create a new superuser and disable the default *cassandra* user. To do so:
+
+1. Log in with the *cassandra* user and execute the following query:
+
+   `CREATE ROLE <new_super_user> WITH PASSWORD = "<STRONG PASSWORD>" AND SUPERUSER = true AND LOGIN = true;`
+
+1. Switch to your *new_super_user* and execute the following query:
+
+   `ALTER ROLE cassandra WITH SUPERUSER = false AND LOGIN = false;`
+
+1. Applying the principle of separation of privileges, we recommend that you also create a dedicated user for DataMiner:
+
+   `CREATE ROLE dataminer WITH PASSWORD '<STRONG PASSWORD>' AND LOGIN = true;`
+
+1. Finally, set the new credentials in DataMiner Cube. For more information, see [Configuring the database settings in Cube](xref:Configuring_the_database_settings_in_Cube).
+
+1. You may need to **restart DataMiner** for the changes to take effect.
+
+> [!NOTE]
+> You can also configure Cassandra to use an LDAP server for authentication. However, this is beyond the scope of this guide.
+
+### Enabling the *CassandraAuthorizer*
+
+Implementing a Zero Trust architecture requires applying the principle of least privilege across your infrastructure. This is especially true for your databases. By default, DataMiner installs Cassandra with authorizer **AllowAllAuthorizer**, effectively granting all permissions to all roles. However, Cassandra also supports the **CassandraAuthorizer**, which allows more granular permission management.
+
+To enable the *CassandraAuthorizer*:
```

Comment:
Could you add a bit more detail to this procedure? As it is, it might not be clear for everyone.